external/fv3fit/fv3fit/data/tfdataset.py

```diff
@@ -8,6 +8,26 @@
 import tempfile
 import xarray as xr
 import numpy as np
+from vcm import safe
+
+
+SAMPLE_DIM_NAME = "_fv3fit_sample"
+
+
+def stack(ds: xr.Dataset, unstacked_dims: Sequence[str]):
```

Comment:
I didn't want to share the implementation of stacking here from any central location, as implementation details like the sample dimension being the first dimension are important here.